### ISSUE TITLE
[BUG] Corruption of overlay/vr menu (desktop/VR View completely corrupted) and stuttering

### ISSUE THREAD

**Issue by panmarco83** [ValveSoftware/SteamVR-for-Linux#512]
**Bug description**
- Stuttering even on default landing space. (Probably related to the corruption)
- Flashing around right eye display (probably in the blacked out area around what is visible and related to the corruption)
- VR Overlay/Menu is badly corrupted. (Screenshot nr. 1)
- VRView is completely corrupted, you can barely see the outlines of the landing space in the corrupted image. (Screenshot nr. 2)

**To Reproduce**
- Install steam, install steamVR, and then run as one would normally do

**Expected behavior**
- Not stuttering even on the basic-default landing space.
- VR Menu/Overlay should not be corrupted.
- VR View / 2D-Desktop view of HMD should not be corrupted.

**System Information (please complete the following information):**
- Distribution: Arch rolling
- SteamVR version: 1.21.12 (Have tried in Beta and linux_v1.14, but still had stuttering and corruption to a slightly different degree)
- Steam client version: "Built: Mar 14 2022, at 19:49:17, Steam package version: 1647446817"
- Opted into Steam client beta?: No
- Graphics driver version:  2022.Q2.1 or Mesa 22.0.1
- Gist for SteamVR System Information: [[SysInfo Gist](https://gist.github.com/panmarco83/29455ec1a01e13340f550007f839b933)]

**Screenshots**
Nr. 1
![IMG_20220425_215750](https://user-images.githubusercontent.com/67251243/165165850-02e08c0d-4cdd-4f65-95a3-79aa1dfdec53.jpg)

Nr. 2
![Screenshot at 2022-04-25 21-56-15](https://user-images.githubusercontent.com/67251243/165166182-c339af93-35c7-416f-ac53-33629b109c02.png)

**Additional context**
- The Vive Pro that is having these problems was purchased in the last week since the old Vive 1-Gen was showing the same problems (except the VR 2D View) so I thought it might be that it was dying since I've had it for almost 6 years now without replacing anything other than the power adapter to one of the base stations that died years ago.
- Also, I can run VR programs and games (except for the room setup that Segfaults on launch so I couldn't map out the usable space other than using the quick calibrate method) more or less well except for excessive stuttering/lag in some cases, even VRChat worked more or less okay to an extent.

**Note:** Commenters who are also experiencing this issue are encouraged to include the "System Information" section in their replies.

**Comment by ChristophHaag** [comment]
> GPU Vendor: "AMD Radeon RX 6800 XT" GPU Driver: "8388829"

SteamVR only works correctly on radv, not amdvlk or amdgpu-pro.

**Comment by panmarco83** [comment]
@ChristophHaag that would explain a bit, I don't know what was default with Arch rolling at install but I wouldn't mind a pointer where I can read up on how to cleaning out everything else so it's only RADV

**Comment by panmarco83** [comment]
Okay, I wish there was a warning at launch that would warn about this.
After following [[this]](https://wiki.archlinux.org/title/Vulkan#Selecting_vulkan_driver) since I'm on arch linux, it seems at least most of the corruption is gone, BUT the right eye is still getting a bright halo around the view area.
On the vive pro, I can't remove the lsense (See [[this]](https://www.ifixit.com/Teardown/HTC+Vive+Pro+Teardown/106064) iFixIt teardown) because it is mounted directly to the screen so I can't get a good look at what sort of distortion that is present around the right eye.

**Comment by kisak-valve** [comment]
Hello @panmarco83, sounds like you've encountered https://github.com/ValveSoftware/SteamVR-for-Linux/issues/480. Try setting SteamVR's launch options to `RADV_DEBUG=zerovram %command%` as a workaround.

**Comment by panmarco83** [comment]
@kisak-valve that worked, thank you !
There are still a few quirks that are annoying, like the fact that the base stations still can't be controlled by steamvr itself and that a "thirdparty" program/app/script (steam_utils) is needed to control them but that is outside the scope of this bug report.
On the other hand, I haven't tried the room-setup program again, but I have a feeling it will still SegFault again unless it was specifically because of the RADV driver not being used.
As well as after managing to switch to RADV, I tried to run a few programs and games but most would launch, get to the first 3D/VR frame, then crash out. very odd.
BUT for some reason, the first one I tried was VRChat, and it ran for about 1 minute until it crashed.
Confusing ..
